src/infra/postgres/client/client.go

```diff
@@ -80,19 +84,25 @@ func (c *PostgresClient) SelectDeleted(ctx context.Context, model ...interface{}
 }
 
 func (c *PostgresClient) UpdatePK(ctx context.Context, model ...interface{}) error {
-	_, err := c.db.ModelContext(ctx, model...).WherePK().Update()
+	res, err := c.db.ModelContext(ctx, model...).WherePK().Update()
 	if err != nil {
 		return parseErrorResponse(err)
 	}
+	if res.RowsAffected() != 1 {
```

Comment:
An update operation does not return "not found" if the update doesn't update anything. If the update does not update anything, it's still a success